README.md

```diff
@@ -1,16 +1,11 @@
 # The JFK Files
-Explore the JFK Assassination files using the Cognitive Search features of Azure Search.
-You can watch the demo in action in a short [online video](https://channel9.msdn.com/Shows/AI-Show/Using-Cognitive-Search-to-Understand-the-JFK-Documents)
-or explore the JFK files yourself with our [online demo](https://aka.ms/jfkfiles-demo).
+Explore the JFK Assassination files using the Cognitive Search features of Azure Search. You can watch the demo in action in a short [online video](https://channel9.msdn.com/Shows/AI-Show/Using-Cognitive-Search-to-Understand-the-JFK-Documents) or explore the JFK files yourself with our [online demo](https://aka.ms/jfkfiles-demo).
 
 ## Cognitive Search - An AI-first approach to content understanding
 This project demonstrates how you can use both the built-in and custom AI in Cognitive Search. Cognitive Search ingests your data from almost any datasource and enriches it using a set of cognitive skills that extracts knowledge and then lets you explore the data using Search.
 
 ![JFK files Cognitive Search](images/jfk-cognitive-search.jpg)
 
-
-
-
 ## JFK Files Architecture
 The JFK files example leverages the built-in Cognitive Skills inside of Azure Search and combines it with custom skills using extensibility.  The architecture below showcases how the new Cognitive Search capabilities of Azure enable you to easily create structure from almost any datasource.
```

Comment:
"inside of Azure Cognitive Search" or "inside of Cognitive Search" could work when referring to the product.

Reply:
I know it wasn't ment to be part of this PR, but I updated that text to match this suggestion as well.